### PR TITLE
feat(blog): add reworked element selection post

### PR DESCRIPTION
**Pull request by nikku**
A post mortem blog publishing our recent selection improvements.

image

### CODE REVIEW

File: src/blog/2023/010-reworked-element-selection.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+releases:
+  - 'bpmn-js@15.0.0'
+  - 'dmn-js@14.7.0'
+  - 'diagram-js@12.6.0'
+
+---
+
+<p class="introduction">
+  The latest releases ship of our [BPMN](https://github.com/bpmn-io/bpmn-js) and [DMN](https://github.com/bpmn-io/bpmn-js) toolkits ship with a reworked selection UI. That improved UI makes diagram element selection more ergonomic and less verbose.
```
Comment (barmac) [inline src/blog/2023/010-reworked-element-selection.md:21]: DMN link should point to https://github.com/bpmn-io/dmn-js

File: src/blog/2023/010-reworked-element-selection.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+</p>
+
+<!-- continue -->
+
+Selection is an integral part of the core modeling UX. It has to work; you come across it immediately, and it leaves a lasting impression. Recent releases of our modeling toolkit simplify selection following the selected element's shape. We think the change results in a less verbose look and feel and clearly distinguishes single-element selections and multi-selection:
```
Comment (barmac) [inline src/blog/2023/010-reworked-element-selection.md:26]: "We think" could be removed. It weakens the message.

File: src/blog/2023/010-reworked-element-selection.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/s/start">
+    <img src="{{ assets }}/attachments/blog/2023/010-selection-rework.gif" alt="Improved selection in BPMN">
```
Comment (barmac) [inline src/blog/2023/010-reworked-element-selection.md:30]: 🥇  for `alt`

File: src/blog/2023/010-reworked-element-selection.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    Enjoy reworked selection in BPMN and DMN editing.
+  </p>
+</div>
+
+Try it on [demo.bpmn.io](https://demo.bpmn.io/s/start).
+
+
+## Wrapping up
+
+As users of our [demo](https://demo.bpmn.io) may have noticed, the reworked popup menu is only the first step on our journey to incorporate [create/append anything](https://bpmn-io.github.io/bpmn-js-connectors-extension/?aa=1) into our core. Stay tuned for future exciting updates in core modeling.
```
Comment (barmac) [inline src/blog/2023/010-reworked-element-selection.md:43]: Looks like a WIP copy-paste.

Comment (nikku) [reply src/blog/2023/010-reworked-element-selection.md:43]: I'll fix it :).

Comment (nikku) [reply src/blog/2023/010-reworked-element-selection.md]: Fixed via https://github.com/bpmn-io/bpmn.io/pull/182/commits/56bf29cddb8a9744e274238b3cf32fbb5e984280.

File: src/blog/2023/010-reworked-element-selection.md (hunk 2 above)
Comment (nikku) [reply src/blog/2023/010-reworked-element-selection.md]: Addressed via https://github.com/bpmn-io/bpmn.io/pull/182/commits/56bf29cddb8a9744e274238b3cf32fbb5e984280.

**Comment by nikku** [comment]
Ready to be looked at again, and squashed + merged :slightly_smiling_face:.

**Review by barmac — APPROVED** [review]
LGTM

**Pull request merged**